doc/release/release_dev.rst

```diff
@@ -14,50 +14,30 @@ http://scikit-image.org
 
 New Features
 ------------
-- manual segmentation with matplotlib (#2584)
-- hysteresis thresholding in filters (#2665)
-- lookfor function (#2713)
-- montage function (#2626)
-- 2D and 3D segmentation with morphological snakes (#2791)
-- nD support for image moments (#2603)
-- inertia tensor and its eigenvalues can now be computed outside of
-  regionprops; available in ``skimage.measure.inertia_tensor`` (#2603)
-- cycle-spinning function for approximating shift-invariance by averaging
-  results from a series of spatial shifts (#2647)
-- data generation with random_shapes function (#2773)
-- Haar-like feature (#2848)
-- subset of LFW database (#2905)
-- Image moments from coordinate input (#2859)
+
+
 
 
 Improvements
 ------------
-- VisuShrink method for wavelet denoising (#2470)
-- ``skimage.transform.resize`` and ``skimage.transform.rescale`` have a new
-  ``anti_aliasing`` option to avoid aliasing artifacts when down-sampling
-  images (#2802)
-- Support for multichannel images for ``skimage.feature.hog`` (#2870)
-- Non-local means denoising (``denoise_nl_means``) has a new optional
-  parameter, `sigma`, that can be used to specify the noise standard deviation.
-  This enables noise-robust patch distance estimation. (#2890)
-- New ``alignment`` parameter in ``skimage.feature.plot_matches``. (#2955)
+
+
 
 
 API Changes
 -----------
-- ``skimage.util.montage.montage2d`` is now available as
-  ``skimage.util.montage2d``.
-- ``skimage.morphology.binary_erosion`` now uses ``True`` as border
-  value, and is now consistent with ``skimage.morphology.erosion``.
+
+
+
 
 Deprecations
 ------------
-- ``skimage.util.montage2d`` is deprecated and will be removed in 0.15.
-  Use ``skimage.util.montage`` instead.
+
+- Python 2 support has been dropped. Minimal required Python version is 3.4.
```

Comment:
Python 2.7

Reply:
Minimal required Python version (has been?) raised to 3.4.

Reply:
2 is ok here: we want to be clear that *no* 2.x version is supported.

Anyway, this statement is too confusing for those that don't know what "deprecation" means technically. I would say, "Python 2 support has been dropped in the development version. Users of the development version should have Python >=3.5". Anyway I want to have the minimum 3.x debate (3.5 vs 3.6; 3.4 is out of the question) after release so it would be best to leave this unspecified.